### PR TITLE
feat: add new icons for campaing

### PR DESCRIPTION
**Pull request by GusBaamonde**
- update build version

new-icon

### CODE REVIEW

**Comment by GusBaamonde** [comment]
test this please

**Comment by cbernat** [comment]
test this please

File: src/components/Reports/CampaignsHistory/CampaignsHistory.js [hunk 1]
```diff
@@ -77,65 +77,64 @@ const CampaignsHistory = ({ location, dependencies: { dopplerApiClient } }) => {
...
+              {/* the style it's temporal because there is a bug in the styles */}
+              <span className={getSubscriberStatusCssClassName(state.subscriber.status)}></span>
+              <FormattedMessage id={'subscriber.status.' + state.subscriber.status} />
+            </span>
+          </div>
+          <div className="col-sm-12 dp-block-wlp">
+            <div className="dp-table-responsive">
+              <table
+                className="dp-c-table"
+                aria-label="Resultado de historial de suscriptores"
```
Comment (cbernat) [inline src/components/Reports/CampaignsHistory/CampaignsHistory.js]: I would love this texts were internationalized. What do you think? It's a bit beyond the scope of this change, perhaps in another commit.

File: src/components/Reports/CampaignsHistory/CampaignsHistory.js [hunk 2]
```diff
@@ -17,31 +17,31 @@ const CampaignsHistory = ({ location, dependencies: { dopplerApiClient } }) => {
     let subscriberCssClass = '';
     switch (status) {
       case 'active':
-        subscriberCssClass = 'user--active';
+        subscriberCssClass = 'ms-icon icon-user user--active';
```
Comment (fcoronelmakingsense) [inline src/components/Reports/CampaignsHistory/CampaignsHistory.js]: ineficient don't need repeat that in all. its always use

**Comment by fcoronelmakingsense** [comment]
remember to apply in the search subscribers grid

**Comment by GusBaamonde** [comment]
Continues in: https://github.com/FromDoppler/doppler-webapp/pull/675

**Pull request closed without merging**